components/Note/src/note.css

```diff
@@ -0,0 +1,49 @@
+.denhaag-note {
+  align-items: flex-start;
+  color: var(--denhaag-note-color);
+  display: flex;
+  flex-direction: row;
+  font-family: var(--denhaag-note-font-family, inherit);
+  font-size: var(--denhaag-note-font-size, 1.125em);
+  font-weight: var(--denhaag-note-font-weight, normal);
+  line-height: var(--denhaag-note-line-height, 1.3);
+  gap: var(--denhaag-note-icon-spacing);
+  padding-block-start: var(--denhaag-note-padding-block, calc(var(--denhaag-note-icon-spacing) / 2));
+  padding-block-end: var(--denhaag-note-padding-block, calc(var(--denhaag-note-icon-spacing) / 2));
+}
+
+.denhaag-note--info {
+  --denhaag-note-icon-color: var(--denhaag-note-icon-info-color, #ffffff);
+  --denhaag-note-icon-fill: var(--denhaag-note-icon-info-fill);
+  --denhaag-note-icon-stroke: var(--denhaag-note-icon-info-stroke, var(--denhaag-note-icon-info-fill));
+}
+
+.denhaag-note--warning {
+  --denhaag-note-icon-color: var(--denhaag-note-icon-warning-color, #ffffff);
+  --denhaag-note-icon-fill: var(--denhaag-note-icon-warning-fill);
+  --denhaag-note-icon-stroke: var(--denhaag-note-icon-warning-stroke, var(--denhaag-note-icon-warning-fill));
+}
```

Comment:
@robbieatacato I'm not sure about this way of handling icons in this component. In other components such as the Alert, if a component influences the color of an icon, it only changes the color, not other SVG properties. I don't think we should assume an icon will look appropriate if we change its color, fill and stroke values.

Reply:
@robbieatacato I have to agree with @bddjong here. Being able to change the `color` is very useful and I would definitely keep that functionality as there is no real way that can go wrong and it adds a lot of customizability, but I would try to stay clear of other properties as this is very dependent on icon usage and can lead to some unintended side effects.

Reply:
@robbieatacato Conclusion here is to keep `--denhaag-note-icon-color` to allow users to set a color for an their SVG. Other SVG properties such as fill and stroke won't be changed.

Reply:
I re-added `--denhaag-note-icon-color` in commit: https://github.com/nl-design-system/denhaag/pull/611/commits/1b1c28327b0f64cfbdfdc9bdb73df890dc5eb2f5